docs/content/modules/tensor_mechanics/systems/Materials/RecomputeRadialReturn.md

```diff
@@ -1,11 +1,78 @@
+$$
+\epsilon_{total} = \epsilon_{mechanical} + \epsilon_{eigenstrain}
+= \left( \epsilon_{elastic} + \epsilon_{inelastic} \right) + \epsilon_{eigenstrain}
+= \epsilon_{elastic} + \left( \epsilon_{plastic} + \epsilon_{creep} + \epsilon_{damage}  \right) + \epsilon_{eigenstrain}
+$$
+
+The final inelastic strain is returned from the recompute radial material, and `ComputeReturnMappingStress` computes the stress, with a return mapping stress increment following elasticity theory for finite strains. The final stress is calculated from the elastic strain increment.
+$$
+\sigma^{new}_{ij} = C_{ijkl} \left( \Delta \epsilon^{elastic}_{kl} + \epsilon^{old-elastic}_{kl} \right)
+$$
+
+When more than one radial recompute material is included in the simulation, as in Combined Power Law Creep and Linear Strain Hardening, `ComputeReturnMappingStress` will iterate over the change in the calculated stress until the return stress has reached a stable value.  
+
+Users can print out any of these strains and stresses using the `RankTwoAux` as described on the
+[Introduction/Visualizing Tensors](auto::/introduction/VisualizingTensors) page.
+
+##Writing a New Recompute Material
+New radial return models must inherit from `RecomputeRadialReturn` and must overwrite the six virtual methods.  
+
+  - **initQpStatefulProperties**: Set the initial values for all new material properties that are not initialized by an input parameter; generally the material properties initialized in this method are all set to zero.
+  - **computeStressInitialize**: Calculate the initial trial stress state, the yield surface value, and any hardening or softening parameters at the start of the simulation time increment.
+  - **computeResidual**: In each iteration over the inelastic strain increment, calculate the value of the effective scalar trial stress subtracted by the yield surface function.
+  - **computeDerivative**: In each iteration over the inelastic strain increment, calculate the derivative of the yield surface function with respect to the inelastic strain increment.
+
+  - **iterationFinalize**: Store the value of the inelastic strain increment at the end of each iteration.
+  - **computeStressFinalize**: Update the stress after convergence on the inelastic strain increment has been reached.
+
+Additionally, new radial return methods must also overwrite a single method from the MOOSE `Material` class.
+
+  - **resetQpProperties**: Set the material property used in the iteration, usually $ \Delta p $, to zero at the start the iteration.  This method is necessary to avoid incorrect material property values.
 
-!parameters /Materials/RecomputeRadialReturn
+More details on how to write the equivalent yield surface equation for a creep model are given in Dunne and Petrinic.
 
-!inputfiles /Materials/RecomputeRadialReturn
 
 !childobjects /Materials/RecomputeRadialReturn
```

Comment:
This particular class is not used in any input files, so I've deleted the input file link in just this file.  The other classes are showing the input files nicely though